src/app/shared/horizontal-carousel/horizontal-carousel.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {HorizontalCarousel, HorizontalCarouselModule} from './horizontal-carousel';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {DocsAppTestingModule} from '../../testing/testing-module';
+
+describe('HorizontalCarousel', () => {
+  let fixture: ComponentFixture<HorizontalCarousel>;
+  let component: HorizontalCarousel;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule(
+      {imports: [HorizontalCarouselModule, DocsAppTestingModule]}
+    ).compileComponents();
```

Comment:
These tests pass if I remove `compileComponents` (and the `async`). I think because the components are aot compiled, all the async resources are inlined already so this isn't needed.

// cc @mmalerba